tools/c7n_mailer/c7n_mailer/sqs_message_processor.py

```diff
@@ -178,48 +224,48 @@ def send_message_to_targets(self, targets, data, resources):
                 continue
             self.send_c7n_email(data, email_to_addr, subject, body)
 
-    def send_c7n_email(self, data, email_to_addr, subject, body):
-        email_format = data['action'].get(
-            'template', 'default').endswith('html') and 'Html' or 'Text'
+    def send_smtp_email(self, smtp_server, message):
```

Comment:
The send_c7n_email function was a bit bit, we we broken off the smtp part into it's own dedicated function.